release/src/main/scripts/cut_release_branch.sh

```diff
@@ -139,10 +139,6 @@ if [[ $confirmation != "y" ]]; then
   exit
 fi
 
-git add gradle.properties
-git add sdks/python/apache_beam/version.py
-git add sdks/go/pkg/beam/core/core.go
-git add runners/google-cloud-dataflow-java/build.gradle
```

Comment:
I believe this file could be modified by line 128.

Reply:
Indeed, looks like it is the only one that line affects.